types/undertaker/undertaker-tests.ts

```diff
@@ -34,6 +37,19 @@ const {
     flags,
 } = taker.task("task4").unwrap();
 
+taker.task("task5", () => {
+    const emitter = new EventEmitter();
+    setTimeout(() => emitter.emit("done"), 1000);
+    return emitter;
+});
+
+taker.task("task6", () => exec("ls"));
+
+taker.task("task7", () => {
+    const lsProcess = exec("ls");
+    return fromEvent(lsProcess, "exit");
+});
+
```

Comment:
Since this code is only type checked and not evaluated, there's no need to depend on `rxjs` for this test. You could simply do this instead:

```suggestion
declare const task7: () => import("async-done").Observable;
taker.task("task7", task7);
```


Or, alternatively (to avoid depending on the basic observable in `async-done`):

```suggestion
declare const task7: () => {
    subscribe(next?: (v: any) => void, error?: (e: any) => void, complete?: () => void);
};
taker.task("task7", task7);
```